dask_ml/model_selection/_incremental.py

```diff
@@ -599,20 +613,94 @@ def score(self, X, y=None):
         return self.scorer_(self.best_estimator_, X, y)
 
 
-class IncrementalSearch(BaseIncrementalSearch):
+class AdaptiveStopOnPlateauSearchCV(BaseIncrementalSearchCV):
```

Comment:
Same, this needs to be changed back, or just changed to IncrementalSearchCV.

Reply:
This diff line isn't completely accurate at first glance (`IncrementalSearch` was not renamed to `AdaptiveStopOnPlateauSearchCV`). I added a private class `AdaptiveStopOnPlateauSearchCV` to allow other adaptive algorithms to stop on plateau. I only intend this class to be used Dask-ML developers, not by Dask-ML users.

Reply:
I think that @TomAugspurger 's comment was aimed to try to scope down this PR to just the `info` and `cv_results_` changes so that we can them in.  Any change to the class hierarchy here will probably require more discussion before a good plan is reached.

Reply:
If you have time to separate the changes @stsievert I suspect that that would be appreciated.

Reply:
Ah. Got it. `AdaptiveStopOnPlateauSearchCV` is removed.